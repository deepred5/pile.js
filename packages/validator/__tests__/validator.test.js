import React from 'react';
import { shallow } from 'enzyme';
import Validator from '../src';

describe('Validator suite', () => {
  it('render a validator', () => {
    const wrapper = shallow(<Validator>{()=>{}}</Validator>);
    expect(wrapper).toMatchSnapshot();
  });
});
