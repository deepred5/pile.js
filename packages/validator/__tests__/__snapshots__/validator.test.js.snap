// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Validator suite render a validator 1`] = `
<Fragment>
  <ErrorMessage
    show={false}
  />
</Fragment>
`;
